### ISSUE TITLE
Issue with NoActionbar theme

### ISSUE THREAD

**Issue by DebRockz** [ibrahimsn98/SmoothBottomBar#38]
very cool !! lib.
i am using NoActionBar theme(i.e Theme.AppCompat.Light.NoActionBar / Theme.MaterialComponents.Light.NoActionBar) in this case fragment is not changing according to menu item. In DarkActionBar it works fine.

**Comment by kriticalflare** [comment]
Can confirm , navigation using navigation components breaks when using NoActionBar themes

**Comment by rajam1215** [comment]
@kriticalflare 

Yes @DebRockz is right, with NoActionbar theme, library not works

**Comment by kaustav07** [comment]
Same problem here..cannot use it with navigation components as my theme is of NoActionBar

**Comment by mohammadbabaeitb** [comment]
@DebRockz library does not work with NoActionbar theme, but you can hide the action bar
`  supportActionBar!!.hide()
`

**Comment by ekifimo** [comment]
hello @ibrahimsn98 how to fix
`bottomBar.setupWithNavController(menu,navController)` ?

**Comment by w1rya** [comment]
I experience this issue too

**Comment by OEssHRERLr770rGGLOH0** [comment]
Same here! NoActionbar themes break navigation.

**Comment by johnmbiya** [comment]
please i have the same issue
not working without actionBar
